### DISCUSSION TITLE
How to turn off flymake globally

### DISCUSSION THREAD

**Discussion by donneyluck** [abougouffa/minemacs#162] [Q&A] [answered]
When I edit a c++ file, eglot will be turned on by default and flymake will be turned on automatically. 
I need to manually close flymake every time. 
I have checked the code and found no hooks between eglot and flymake. How should I close it?

**Comment by stormwatch** [comment, marked as answer]
See https://github.com/joaotavora/eglot/issues/123#issuecomment-444104870

**Reply by donneyluck** [reply]
thks
